Problem2.java

```diff
@@ -0,0 +1,51 @@
+//3 Sum
+// Time Complexity :
+// Space Complexity :
+// Did this code successfully run on Leetcode :
+// Any problem you faced while coding this :
+//3 pointers, low, high & curr(i)
+// we use a for loop to travers each element and then run two sum in remaining element to check the 3sum. 
+// if we have sum < 0 , update left pointer, if we have sum >0, update right ppointer.
+//for checking duplicate, while loop, & increase low pointer till we get different element
+
```

Comment:
include time and space complexity

Reply:
// Time Complexity : O(n^2)
// Space Complexity :O(1) Not considering the result list